onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, model, per_channel, reduce_range, mode, static, weight_qType,
         self.static = static  # use static quantization for inputs.
         self.fuse_dynamic_quant = False
         self.extra_options = extra_options if extra_options is not None else {}
+        self.quantize_const_node_only = 'ConstOnly' not in self.extra_options or self.extra_options['ConstOnly']
```

Comment:
>quantize_const_node_only [](http://example.com/codeflow?start=13&length=24)

As we synced, MatMulBConstOnly looks like a better name

Reply:
The name is changed.